.circleci/docker/common/install_base.sh

```diff
@@ -2,17 +2,10 @@
 
 set -ex
 
-if [[ "$UBUNTU_VERSION" == "14.04" ]]; then
-  # cmake 2 is too old
-  cmake3=cmake3
-else
-  cmake3=cmake
-fi
-
-if [[ "$UBUNTU_VERSION" == "18.04" ]]; then
+if [[ "$UBUNTU_VERSION" == "18.04"* ]]; then
```

Comment:
Can you please add a comment explaining why this is necessary?

Reply:
added